draft-fossati-tls-attestation.md

```diff
@@ -206,22 +210,45 @@ Among other security metrics, the PAT contains evidence about the integrity of a
 
 The protocol's security relies on the verifiable binding between these two logically separate units of evidence.
 
+## Authentication vs. Attestation
+
+As noted, the protocol supports either combined platform attestation with X.509 certificate authentication, or attestation only.
+
+Attestation when used alone is vulnerable to identity spoofing attacks, in particular when zero-day attacks exist for a class of hardware. (TODO: reference). Therefore it needs to be combined with traditional authentication, which in the case of TLS takes the form of X.509 certificates.
+
+We RECOMMEND that regular applications only use the combined mode, which provides the full security guarantees of an authenticated TLS handshake (for the peer/peers being authenticated) as
+well as guarantees on platform integrity.
+
+The attestation-only mode is included in this document for specialized use cases, including initial provisioning of the TLS stack. In these cases, additional security controls SHOULD be provided,
+such as hardware-enforced time limitations, or use of platform-level APIs in the case of cloud infrastructure.
+
+## TLS Identity Key (TIK) {#tik}
+
+A central property of this protocol is that it binds a peer's identity key, the TIK, to the TLS handshake.
+This applies to the client's identity (TIK-C), the server's identity (TIK-S) or both.
+
+In the standard mode that combines attestation with certificate authentication, the TIK MUST be
+the X.509 certificate's end entity key.
+
+In the attestation-only mode, the TIK is generated by the TEE and is only used for TLS handshakes.
+It may be used by (specialized) applications to provide continuity of identity between TLS sessions with the same peer. 
+
 # Attestation Extensions
 
 As typical with new features in TLS, the client indicates support for the new
 extension in the ClientHello message. The newly introduced extensions allow
 remote attestation credentials and nonces to be exchanged. The nonces are used
 for guaranteeing freshness of the exchanged evidence when the background check
-model is in use.
+model is in use. Nonces are not used in the passport model, because the expectation
+of freshness is more relaxed and is only governed by the lifetime of the signed
+attestation results.
 
-When either the evidence or the attestation results extension is successfully
+When either the Evidence or the Attestation Results extension is successfully
 negotiated, the content of the corresponding Certificate message contains a
 payload that is encoded based on the wrapper defined in {{-cmw}}.
 Both JSON and CBOR serializations are allowed in CMW, with the emitter choosing which serialization to use.
 
-In TLS a client has to demonstrate possession of the private key via the
-CertificateVerify message, when client-based authentication is requested. The
-attestation payload must contain assertions relating to the client's TLS
+The attestation payload must contain assertions relating to the client's TLS
```

Comment:
I see it as a strong requirement for the attestation result producer.
Therefore, I’d prefer an upper-case must.